index.html

```diff
@@ -562,10 +562,14 @@ <h3>
         </p>
       </section>
     </section>
-    <section>
+    <section class="atrisk">
```

Comment:
Has there been any more discussion about the "at risk" status of this feature? I don't think we can just deprecate it at this point; too many sites rely on it (though as I say below, I think it's fine for it to be perpetually optional).

Reply:
I don‘t have a strong feeling on "at risk". Keeping it optional also leaves the door open for browsers to decide to never do anything with it or to remove support is they have it an are unhappy with it. If we get to the point that no UA is interested in supporting it anymore, we can label it "at risk" and begin to let folks know they may want to consider dropping it from their code.